stix_shifter_utils/stix_transmission/utils/RestApiClient.py

```diff
@@ -82,13 +83,16 @@ def __init__(self, host, port=None, headers={}, url_modifier_function=None, cert
     # This method is used to set up an HTTP request and send it to the server
     def call_api(self, endpoint, method, headers=None, data=None, urldata=None, timeout=None):
         try:
-            # covnert server cert to file
+            # convert server cert to file
             if self.server_cert_file_content_exists is True:
-                with open(self.server_cert_name, 'w') as f:
-                    try:
+                cert_bundle = os.environ.get('REQUESTS_CA_BUNDLE')
+                try:
+                    if cert_bundle:
+                        shutil.copy(cert_bundle, self.server_cert_name)
+                    with open(self.server_cert_name, 'a') as f:
```

Comment:
why mode 'a' instead of 'w' for file opening? 'a' means append to whatever there is?